Gemspec

```diff
@@ -0,0 +1 @@
+../plugins/openproject-xx_test
```

Comment:
Please remove this reference

Reply:
Oops. Missed this one when committed. Removed in following push.